lib/schemes/HttpUriPlugin.js

```diff
@@ -19,6 +19,37 @@ const memoize = require("../util/memoize");
 
 const getHttp = memoize(() => require("http"));
 const getHttps = memoize(() => require("https"));
+const proxyFetch = (request, proxy) => (url, options, callback) =>
+	Promise.resolve(
+		proxy &&
+			new Promise((resolve, reject) => {
+				const { hostname: host, port } = new URL(proxy);
+
+				getHttp()
+					.request({
+						host, // IP address of proxy server
+						port, // port of proxy server
+						method: "CONNECT",
+						path: "raw.githubusercontent.com:80"
```

Comment:
hardcoded path?

Reply:
Due to the following case where the URL has been abandoned via GFW.

```js
import codeOfConduct2 from "https://raw.githubusercontent.com/webpack/webpack/main/CODE_OF_CONDUCT.md";
```